projects/cashmere/src/lib/tabs/tab-set/tab-set.component.ts

```diff
@@ -257,16 +256,31 @@ export class TabSetComponent implements AfterContentInit {
 
     tabArrowInterval = interval(100);
 
+    _handleTabsWheel(event: WheelEvent): void {
+        const scrollDirection = Math.abs(event.deltaX) > Math.abs(event.deltaY) ? Math.sign(event.deltaX) : Math.sign(event.deltaY);
+        const scrollDistance = Math.sqrt(Math.pow(event.deltaX, 2) + Math.pow(event.deltaY, 2)) * scrollDirection;
```

Comment:
I am sure Pythagoras would be thrilled to know we are still using his theorem, lol